frontend/wallet/src/render/Theme.re

```diff
@@ -20,7 +20,8 @@ module Colors = {
 
   let bgColor = white;
 
-  let bgColorElectronWindow = "#00E9E9E9";
+  // Linux doesn't support transparency
+  let bgColorElectronWindow = "#FFE9E9E9";
```

Comment:
Do we want to do transparency on macOS only? Seems fine to be inconsistent to me.

Reply:
We don't use the transparency on mac, it was an artifact from an old build.

Reply:
The comment was just there to remember in the future if we add back transparency. In general the transparency support seems a little buggy regardless of platform.